pkgs/racket-doc/scribblings/reference/eval-model.scrbl

```diff
@@ -29,6 +30,13 @@ to switch to the usual langle/rangle that is used in syntax definitions.
    (syntax-rules () [(_ a b) (*state (racket a) (racket b))]))
 ;}
 
+@;{
+I worked on sections 1--9. More work to come on them and the remaining
+sections.
+
+~~Paul C. Anagnostopoulos
+}
+
```

Comment:
You can remove this, too. GitHub is the right place for this kind of commentary, and the Git history generally records who worked on what.